### DISCUSSION TITLE
org-babel support

### DISCUSSION THREAD

**Discussion by edmundmiller** [karthink/gptel#364] [Ideas]
I was looking at https://github.com/tyler-dodge/org-assistant and I thought it would be great to have an org babel package. Something like

```
#+BEGIN_SRC gptel :model kagi:summarize:agnes :system-message "Summarize this link"
https://blah.com
#+END_SRC
```

**Comment by karthink** [comment]
Sure, you can write an ob-gptel library/package quite easily using gptel's gptel-request API.

(gptel-request handles the plumbing that gptel-send and the transient menu use.  There are examples of how to use it in the wiki.)
